server/models/testing/section-relation.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const sectionRelationSchema = new Schema({
+  parentId: {
+    type: Schema.Types.ObjectId, index: true, ref: 'Parent',
```

Comment:
If you look [here](http://mongoosejs.com/docs/populate.html), you will see, that:
> The ref option is what tells Mongoose which model to use during population

In our case you don't have a model named 'Parent'. The model, that should be used in there is 'Section'.